code/__DEFINES/DNA.dm

```diff
@@ -166,7 +166,7 @@
 
 //organ defines
 #define STANDARD_ORGAN_THRESHOLD 100
-#define STANDARD_ORGAN_HEALING 0.0005
+#define STANDARD_ORGAN_HEALING 0.00050
```

Comment:
?

Reply:
this way it's clearer to see that it's 50/100 000, not 5/100 000. i was comparing that value with STANDARD_ORGAN_DECAY 0.00111 and at first glance i thought the ratio of decay to healing was 111/5, but in reality it's 111/50

Reply:
Can you just put in `(50 / 100000)` then? This isn't super readable.